Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift

```diff
@@ -21,12 +21,11 @@ class RubricCircleView: UIView {
     private static let w: CGFloat = 49
     private static let space: CGFloat = 10
     private var buttons: [UIButton] = []
-    var rubric: RubricViewModel? {
```

Comment:
The `frame` was not correct when this was being run and thus the circles were being rendered off the screen. So doing it in `layoutSubviews` ensures the frame will be accurate when `setupButtons` is called.